### ISSUE TITLE
Navbar buttons & search field do not collapse

### ISSUE THREAD

**Issue by firechhe** [giterlizzi/dokuwiki-template-bootstrap3#537]
### Description

If I open the dokuwiki with bootstrap template my navbar is display in a "two line mode". The width is not small enough to collapse the navbar.

### Steps to reproduce

**Expected behavior:** The navbar should collapse to prevent that the navbar displays in two lines. 

**Actual behavior:** The navbar first collapses if I make the browser window very small. On a iPad Air it is not possible, so I have this two line navbar in vertical alignment.

### Versions

* [Bootstrap3 Template] 2021-03-11
* [DokuWiki] Release 2018-04-22c "Greebo"
* [Plugins]
* [PHP] 7.4
* [Browser] Firefox (Windows); Firefox (iOS); Safari (iOS)

### Screenshots or Logs

![01](https://user-images.githubusercontent.com/43295016/112903909-74b89b80-90e8-11eb-9317-70c340de16b1.jpg)
![02](https://user-images.githubusercontent.com/43295016/112903911-75513200-90e8-11eb-9325-df87402b8148.jpg)

**Comment by phy25** [comment]
Bootstrap won't collapse based on the navbar item length, but based on the viewport width. I don't think it's easy to tweak that viewport width logic without recompiling Bootstrap.

**Comment by firechhe** [comment]
I can not believe that. Bootstrap is a mobile first template and the navbar looks like this? The problem is also that the last viewed pages are hidden from the navbar, so you can not navigate back do last viewed pages....

**Comment by aubreyz** [comment]
> Bootstrap won't collapse based on the navbar item length, but based on the viewport width. I don't think it's easy to tweak that viewport width logic without recompiling Bootstrap.

Not really. The coder can say what screen width corresponds to what type of toolbar. What is that shaded area to the left - I am not using the current version but you might have confounded the coder's intent by adding too much to the toolbar.

**Comment by firechhe** [comment]
The shaded area is the title of the wiki. I have shortened it and it has been improved. But on a iPad the navbar is still in two lines. 
I can not really disable links or buttons of my navigation. 

Maybe I can adjust the value at which screen width the toolbar collapsed. But I dont know where I could find this value?
